src/widgets/universedisplay.cpp

```diff
@@ -117,7 +157,7 @@ void UniverseDisplay::setFlickerFinder(bool on)
         {
             m_flickerFinderLevels[i] = m_listener->mergedLevels().at(i).level;
             m_flickerFinderHasChanged[i] = false;
-            setCellColor(i, Qt::white);
+            setCellColor(i, Preferences::Instance().GetTheme() == Themes::LIGHT ? Qt::white : Qt::black);
```

Comment:
Ideally this would be a switch statement, with no default, as we _might_ add more themes later.